spec/support/fill_in_the_form_steps.rb

```diff
@@ -63,6 +63,16 @@ def and_has_given_their_name
     end
   end
 
+  def and_has_given_the_minimal_name_information
+    expect(page).to have_content(I18n.t("coronavirus_form.questions.name.title"))
+    within find(".govuk-main-wrapper") do
+      fill_in "first_name", with: "Test Please Ignore"
```

Comment:
We can leave the text as it is, but just incase you didn't know, we don't write any of the smoke tests to the database thanks to [this line](https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form/blob/9fd651d30f9cd06074bcedfb4cf0e9f21b80d9d8/spec/spec_helper.rb#L25).